### DISCUSSION TITLE
Sync when content is unwatched?

### DISCUSSION THREAD

**Discussion by furiousM** [luigi311/JellyPlex-Watched#197] [Q&A]
When I 'unwatch' a content in one server, once jellyplex-watched runs it seems to only look at 'watched' states and if it's watched on any server it will set them all to 'watched'. So to 'unwatch' something I have to set both servers to unwatched before jellyplex runs. Is this correct or am I missing something?
Would it be possible to look at the latest change, be it to watched or unwatched and sync across?

**Comment by luigi311** [comment]
I dont think theres a way for me to keep track of something like that easily so to try and keep it as simple as possible i only work based on if a media is marked as watched. There might be a way for me to handle it but it would complicate the logic so ive been avoiding it since going though the code of this is already confusing enough as it is.

Whenever i do need to unmark something i do have to do it on all my servers on my side before the next loop occurs.
